### PR TITLE
fail fast when building vhd.

### PR DESCRIPTION
**Pull request by andyliuliming**

### CODE REVIEW

**Comment by paulgmiller** [comment]
We probably need to the bad calico images before this will pass but definitely think this is a good idea. @xuto2

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -1,4 +1,9 @@
 #!/bin/bash
+set -e -o pipefail
+
+echo "##### debug starts"
+cat /home/packer/provision_installs.sh
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:5]: This file is pretty static, why wouldn't we have it in build VHD?

If so, How about this
cat /home/packer/provision_installs.sh > /dev/null

and do the same for all the below files?

File: pkg/agent/utils.go [hunk 2]
```diff
@@ -243,6 +243,10 @@ func getBase64EncodedGzippedCustomScript(csFilename string, config *datamodel.No
 		// this should never happen and this is a bug
 		panic(fmt.Sprintf("BUG: %s", err.Error()))
 	}
+
+	fmt.Printf("#########\n")
+	fmt.Printf("%s", string(b))
+	fmt.Printf("#########\n")
```
Comment (xuto2) [inline pkg/agent/utils.go:249]: we don't need this in prod

**Comment by xuto2** [comment]
seems not addressing the issue, closing for now.

**Pull request closed without merging**